src/Common/Core/Impl/Security/Credentials.cs

```diff
@@ -18,12 +18,6 @@ public sealed class Credentials : ICredentials {
             return creds;
         }
 
-        public NetworkCredential GetCredential(Uri uri, string authType) {
-#if NETSTANDARD1_6
-            return new NetworkCredential(UserName, Password.ToUnsecureString());
-#else
-            return new NetworkCredential(UserName, Password);
-#endif            
-        }
+        public NetworkCredential GetCredential(Uri uri, string authType) => new NetworkCredential(UserName, Password.ToUnsecureString());
```

Comment:
In `NETSTANDARD2` there will be constructor with `SecureString` - let's not remove this condition.